src/components/comment/index.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { useState } from "react";
```

Comment:
Little nitpick: you can club this into a single line: `import React, {useState} from "react";`